packages/cors-proxy/package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "@kie-tools/cors-proxy",
+  "version": "0.0.0",
+  "description": "",
+  "license": "Apache-2.0",
+  "keywords": [],
+  "homepage": "https://github.com/kiegroup/kie-tools",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kiegroup/kie-tools.git"
+  },
+  "bugs": {
+    "url": "https://github.com/kiegroup/kie-tools/issues"
+  },
+  "main": "dist/index.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build:dev": "rimraf dist && webpack --env dev",
+    "build:prod": "pnpm lint && pnpm test && rimraf dist && webpack",
+    "lint": "run-script-if --bool \"$(build-env linters.run)\" --then \"kie-tools--eslint ./src\"",
+    "setup-dev-env": "cross-env CORS_PROXY_HTTP_PORT=$(build-env corsProxy.dev.port) ",
+    "start": "pnpm build:dev && cross-env CORS_PROXY_HTTP_PORT=$(build-env corsProxy.dev.port) CORS_PROXY_ALLOW_SELF_SIGNED_CERTIFICATES=$(build-env corsProxy.dev.selfSignedCertificates) CORS_PROXY_VERBOSE=$(build-env corsProxy.dev.verbose) node dist/index.js ",
```

I don't think this start command works on windows, you might need to add a powershell script just like we have on other package.json files.

```suggestion
    "start": "run-script-os",
    "start:darwin:linux": "pnpm build:dev && cross-env CORS_PROXY_HTTP_PORT=$(build-env corsProxy.dev.port) CORS_PROXY_ALLOW_SELF_SIGNED_CERTIFICATES=$(build-env corsProxy.dev.selfSignedCertificates) CORS_PROXY_VERBOSE=$(build-env corsProxy.dev.verbose) node dist/index.js",
    "start:win32": "pnpm build:dev && pnpm powershell \"cross-env CORS_PROXY_HTTP_PORT=$(build-env corsProxy.dev.port) CORS_PROXY_ALLOW_SELF_SIGNED_CERTIFICATES=$(build-env corsProxy.dev.selfSignedCertificates) CORS_PROXY_VERBOSE=$(build-env corsProxy.dev.verbose) node dist/index.js\"",
```

---

going to try it thanks!